### ISSUE TITLE
Plotting error when using Jupyter notebook

### ISSUE THREAD

**Issue by andrew-lundgren** [gwpy/gwpy#1012]
As of 0.12.2, there's an error when plotting inline in Jupyter notebooks. It used to be possible to just do `data.plot()`, but that now gives `AttributeError: 'module' object has no attribute 'FigureManagerBase'`.  Example code is:
```
%matplotlib inline
import matplotlib.pyplot as plt
from gwpy.timeseries import TimeSeries
import numpy as np

data = TimeSeries(np.sin(np.linspace(0,6.28*100.,1024)), sample_rate=1024)

data.plot()
```

A workaround is 
```
p1 = plt.figure()
p1.gca().plot(data)
```
but hopefully it's possible to restore the old functionality.

**Comment by duncanmmacleod** [comment]
This is probably a duplicate of #964, fixed by #985. @andrew-lundgren, are you able to install gwpy from the git repo to check that this indeed fixes the issue?

**Comment by duncanmmacleod** [comment]
No response, closing.
